pkg/common/client/oidc/models.go

```diff
@@ -0,0 +1,17 @@
+package oidc
```

Comment:
maybe this should be `auth` package instead of `oidc`, since it seems more general-purpose